src/kernel/arch/x86/gdt.zig

```diff
@@ -438,7 +438,10 @@ pub fn init() void {
     // Load the TSS
     arch.ltr(TSS_OFFSET);
 
-    if (build_options.rt_test) runtimeTests();
+    switch (build_options.test_type) {
+        .NORMAL => runtimeTests(),
```

Comment:
What if it's `ALL_RUNTIME`?

Reply:
The python script will run each test individually, `build_options.test_type` should never be `ALL_RUNTIME` when it gets to the OS code. Maybe could add a check for this. `ALL_RUNTIME` is really only to tell the python runtime to run all the runtime tests.

Reply:
Hmm ok, if it's never supposed to be used by the Zig code I'd prefer if it wasn't part of the enum at all.

Reply:
hmm, idk how to remove it without adding another build option for just running all the runtime tests

Reply:
How about using something similar the previous system where `zig build test` runs the unit tests but adding `-Drt-test=x` will build with runtime testing enabled and can be `normal`, `panic` or `multitasking`? That has the advantage that you can run the kernel with rt-testing enabled and see all the log messages without actually testing. If you want to test all runtime tests you could `&&` each command together in the shell.

Reply:
How about we remove the ALL option and if you want to run all runtime tests you just `&&` the shell commands together?

Reply:
Could do, but the command could get very long if we have lots of tests, also will allow us to easily paralise the the tests if we let the build script do it for us. I don't mind having it in the enum, but would have to add an additional option for running all the runtimes.